content/docs/glossary/muxed-accounts.mdx

```diff
@@ -272,4 +273,81 @@ If we were to make a payment between two muxed accounts that had *different* und
 As is the case for most protocol-level features, you can find more usage examples and inspiration in the relevant test suite for your favorite SDK. For example, [here](https://github.com/stellar/js-stellar-base/blob/master/test/unit/muxed_account_test.js) are some of the JavaScript test cases.
 
 
+## Frequently-Asked Questions
```

Comment:
💭 Each of these sections don't pose a question. If we want this to be a FAQ we should probably pose them as Q and As. e.g. "What happens if a sender sends amounts to a muxed account but the receiver doesn't use muxed accounts?"

Maybe the title shouldn't be FAQ since this doesn't look like a FAQ.

Reply:
Maybe the title should be something like "Common Issues Using Muxed Accounts" or something like that.

Reply:
I've amended the headings to pose questions!